kernels/etherlink/xtask/src/main.rs

```diff
@@ -0,0 +1,302 @@
+// SPDX-FileCopyrightText: 2025 Nomadic Labs <contact@nomadic-labs.com>
+//
+// SPDX-License-Identifier: MIT
+
+use std::env;
+use std::path::Path;
+use std::path::PathBuf;
+
+use anyhow::Context;
+use anyhow::Result;
+use anyhow::bail;
+use clap::Parser;
+use clap::Subcommand;
+use xshell::Shell;
+use xshell::cmd;
+
+const ETHERLINK_DIR: &str = "kernels/etherlink";
+const RISCV_DIR: &str = "src/riscv";
+const SANDBOX_BIN: &str = "riscv-sandbox";
+
+const TX_COUNT: &str = "200";
+const DEFAULT_ROLLUP_ADDRESS: &str = "sr163Lv22CdE8QagCwf48PWDTquk6isQwv57";
+const DEFAULT_INBOX: &str = "src/riscv/assets/etherlink-erc20-inbox.json";
```

Comment:
```suggestion
const DEFAULT_INBOX: &str = format!("{RISCV_DIR}/assets/etherlink-erc20-inbox.json");
```

Reply:
This doesn't work when defining a const, not sure there's a cleaner way than just duplicating the prefix.